conf/sample.config.inc.php

```diff
@@ -33,11 +33,11 @@
  *
  */
 $conf['datasources']['localhost'] = array(
-	'host'	=> 'localhost',
+	'host'	=> getenv('DB_PORT_3306_TCP_ADDR') ?: 'localhost',
```

Comment:
I kinda like the idea of changing this to be more specific. Perhaps something like `ANEMOMETER_DB_PORT`